shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/main/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/registry/status/compute/watcher/ComputeNodeStateChangedWatcher.java

```diff
@@ -70,10 +75,31 @@
         } else if (event.getKey().startsWith(ComputeNode.getOnlineInstanceNodePath())) {
             Optional<InstanceDefinition> instanceDefinition = ComputeNode.getInstanceDefinitionByInstanceOnlinePath(event.getKey());
             return instanceDefinition.isPresent() ? createInstanceEvent(instanceDefinition.get(), event.getType()) : Optional.empty();
+        } else if (event.getKey().startsWith(ComputeNode.getProcessTriggerNodePatch())) {
+            return createShowProcessListTriggerEvent(event);
         }
         return Optional.empty();
     }
     
+    private Optional<GovernanceEvent> createShowProcessListTriggerEvent(final DataChangedEvent event) {
+        Matcher matcher = getShowProcessTriggerMatcher(event);
+        if (!matcher.find()) {
+            return Optional.empty();
+        }
+        if (event.getType() == Type.ADDED) {
```

Comment:
@tuichenchuxin Thank you for your pull request. I think `event.getType() == Type.ADDED` can change to `Type.ADDED == event.getType()`

Reply:
Thank you. Done.